os/tools/configure.sh

```diff
@@ -150,6 +150,12 @@ if [ ! -r "${src_config}" ]; then
   exit 6
 fi
 
+if [ -r ${dest_config} ]; then
```

Comment:
I don't know that this is right. Actually no.
If I configured once without compilation, I can reconfigure. There is no issue. But this change makes error.

How about check ${dest_config} and .version file?
Because .version is made on compilation, existing of it means compiled.
Then, we had better guide "make distclean".

Could you give your opinion?

Reply:
@sunghan-chang,  yes I agree with you. Checking for both ${dest_config} and .version file is necessary. I made changes as per your suggestions. Please check